lib/caesar.rb

```diff
@@ -1,6 +1,18 @@
 class Caesar
-  attr_reader :key
 
-  def initialize
+  def initialize(shift, eng_to_cipher = "the quick brown fox jumps over the lazy dog".to_a)
+    @eng_to_cipher = eng_to_cipher
+    @cipher = eng_to_cipher.chars.rotate(shift).join
   end
-end
\ No newline at end of file
+
+  def encrypt(text)
+    text.tr(@eng_to_cipher, @cipher)
+  end
+
+  def decrypt(text)
+    text.tr(@cipher, @eng_to_cipher)
+  end
+end
+
```

Comment:
From the spec, you should see that the class Caesar has an initialize method which takes no parameters and has a method called eng_to_cipher that takes 2 parameters: the text and the key